README.md

```diff
@@ -67,11 +68,18 @@ This project currently contains two executable packages located in the `cmd` dir
     - reference implementation for using the generated grpc client stubs for
       payment channel API.
 
-For detailed information on the features offered by these two applications and
-steps on how to try them out, see the
+- `perunnodetui` is an app with interactive text based user interface. With
+  visual representation most of the actions, it is easy way to try out payment
+  channel API and understand the perun protocol.
+
+For a tutorial on using perun-node with `perunnodecli`, see the
 [tutorial section](https://labs.hyperledger.org/perun-doc/node/introduction.html#user-guide)
-on the project documentation website. For a tutorial using for `perunnodetui`, see
-[this section](cmd/perunnodetui/Readme.md).
+on the project documentation website.
+
+For a tutorial on using perun-node with `perunnodetui`, see the
+[this section](cmd/perunnodetui/Readme.md). A talk on overview of perun,
```

Comment:
see [this section], remove the "the" in the sentence, then it seems to be fine :)